examples/settings/set-margin.tape

```diff
@@ -0,0 +1,13 @@
+Output "examples/settings/set-margin.gif"
+Set FontSize 20
+Set Width 600
+Set Height 300
+Set Padding 20
+
+# You can also use an image:
+# Set MarginFill "path/to/image.png"
+Set MarginFill "#79B8C3"
+Set MarginSize 20
```

Comment:
I _think_ we'll want to call these `Background` and `Margin` externally. Internally, in the code, we can keep these names.

```suggestion
Set Background "#79B8C3"
Set Margin 20
```

Reply:
Are you sure? I think that the `Background` setting key is a bit misleading, since backgrounds are usually "interior & padding" areas, not "margin" areas.

For example, this is how CSS works ([jsfiddle](https://jsfiddle.net/x3wcuabm/4/)).

Maybe run a poll on discord?

Reply:
Yeah, actually I think you are right, background might suggest the terminal background.

Another approach we could go for is calling it Border, what do you think?

```
Border 20px "#FFF"
```

or 

```
BorderSize 20px
BorderFill "#FFF"
```

Reply:
Perhaps `Margin` is the correct name, Border could be used in conjunction with the Margin I guess.

Reply:
Yeah, I think "Margin" makes the most sense
Border is terminal frame, padding is padding, etc

Just like the CSS box model we're used to!